skops/io/_persist.py

```diff
@@ -28,6 +28,29 @@
 
 
 def save(obj, file):
+    """Save an object using the skops persistence format.
+
+    Skops aims at providing a secure persistence feature that does not rely on
```

Comment:
I think it makes sense to put warnings only to load rather than save, but feel free to ignore if you think it doesn't make sense😅

Reply:
I know I went overboard with warnings, but I really wanted to make sure users don't miss the fact that we're still early in development and know that it's not secure yet. Hopefully, we can fill these gaps soon and then remove the warnings. Until then, I'd rather leave them in.